hiring/remote_test/README.md

```diff
@@ -0,0 +1,16 @@
+# Remote test
+
+This is our current remote test for screening candidates. We provide a candidate with both a brief and a marking guide that clearly explains what we are looking for technically. The purpose of this test is to screen out candidates that cannot complete a set of instructions and perform tasks within a rails app.
```

Comment:
"The purpose of this test is to screen out candidates that cannot complete a set of instructions and perform tasks within a rails app." - I've often found people who apply are not that well versed in Rails. Also think we should be careful about stipulating technologies, when we're trying to broaden our offering. What was your thinking behind this?